plugins/builddep.py

```diff
@@ -135,13 +135,29 @@ def _rpm_dep2reldep_str(rpm_dep):
         return rpm_dep.DNEVR()[2:]
 
     def _install(self, reldep_str):
-        try:
-            self.base.install(reldep_str)
-        except dnf.exceptions.MarkingError:
+        sltr_provides = dnf.selector.Selector(self.base.sack)
+        sltr_provides.set(provides=reldep_str)
+        sltrs = [sltr_provides]
```

Comment:
I don't like the construct with list of selectors. This is not good idea when you can end up with more than one goal per provide.